test/graphql-snapshot.ts

```diff
@@ -177,8 +177,14 @@ function simplifyGraphqlTree(tree: DocumentNode): GraphqlSnapshot {
   return result;
 }
 
-export function makeGraphqlSnapshot(requestBodyStr: string): GraphqlSnapshot {
-  const requestBody = JSON.parse(requestBodyStr);
+interface GraphqlSnapshotInput {
```

Comment:
```suggestion
export interface GraphqlSnapshotInput {
```

It's used in public function declaration